src/components/table/Table.vue

```diff
@@ -265,25 +241,21 @@
                                 />
                             </td>
 
-                            <slot
-                                v-if="$scopedSlots.default"
-                                :row="row"
-                                :index="index"
-                            />
-                            <template v-else>
-                                <BTableColumn
-                                    v-for="column in newColumns"
-                                    v-bind="column"
-                                    :key="column.customKey || column.label"
-                                    internal>
-                                    <span
-                                        v-if="column.renderHtml"
-                                        v-html="getValueByPath(row, column.field)"
+                            <template v-for="(column, index) in visibleColumns">
+
```

Comment:
If you already doing this, remove extra lines everywhere, they're useless there.